test/data-source.test.js

```diff
@@ -103,7 +102,9 @@ describe('DataSource', function() {
         var fn = scope[name];
         var actuallyEnabled = Model.getRemoteMethod(name);
         assert(fn, name + ' should be defined!');
-        assert(actuallyEnabled === isRemoteEnabled, name + ' ' + (isRemoteEnabled ? 'should' : 'should not') + ' be remote enabled');
+        assert(actuallyEnabled === isRemoteEnabled,
+               name + ' ' + (isRemoteEnabled ? 'should' : 'should not') +
+                 ' be remote enabled');
```

Comment:
There are two things in play here: how to indent function arguments when we split function call on mutliple lines, and how to indent multi-line strings.

My current version here

1) Aligns arguments at the same level. 

```js
assert(longArg1,
       argument2ThatIsLongToo,
       arg3);
```

However, in many other places, we put multiple args on one line and simply indent the next line by one level:

```js
assert(longArg1, argument2ThatIsLongToo,
  arg3);
```

or even

```js
foobar(arg1,
  {
     opt1: true,
     opt2: true
  },
  cb);
```

2) For multi-line strings, second and subsequent lines are indented by one level:

```js
var str = 'asd' +
  'line2';
```

@superkhau I find your examples confusing, why is `helloAgain` call indented differently than `hello`, when they are at the same nesting level?

I think the main question here is whether we can live with what's proposed here, or how to change it.

Let's discuss on Slack when you get online.